ci/scenarios/healthcheck/healthcheck_run_test/main.py

```diff
@@ -0,0 +1,138 @@
+
+        :param blocked: was the test blocked by other test?
+        :type blocked: bool
+        :return: results of test
+        :rtype: dict
+        """
+        if not blocked:
+            try:
+                result = HealthCheckCI.validate_healthcheck()
+                return {'status': 'PASSED', 'case_type': HealthCheckCI.CASE_TYPE, 'errors': result}
+            except Exception as ex:
+                HealthCheckCI.LOGGER.error("Healthcheck CI testing failed with error: {0}".format(str(ex)))
+                return {'status': 'FAILED', 'case_type': HealthCheckCI.CASE_TYPE, 'errors': str(ex)}
+        else:
+            return {'status': 'BLOCKED', 'case_type': HealthCheckCI.CASE_TYPE, 'errors': None}
+
+    @staticmethod
+    def validate_healthcheck():
+        """
+        Validate if the healthcheck works
+
+        Will always check: localhost & master
+        Will check if available: slave
+
+        :return:
+        """
+
+        HealthCheckCI.LOGGER.info("Starting to validate the healthcheck")
+
+        storagerouter_master_ips = StoragerouterHelper.get_master_storagerouter_ips()
+        assert len(storagerouter_master_ips) >= 1, "Not enough MASTER storagerouters"
+        storagerouter_slave_ips = StoragerouterHelper.get_slave_storagerouter_ips()
+
+        # setup base information
+        node_ips = [storagerouter_master_ips[0], '127.0.0.1']
+        HealthCheckCI.LOGGER.info("Added master `{0}` & `127.0.0.1` to the nodes to be tested by the healthcheck"
+                                  .format(storagerouter_master_ips[0]))
+
+        # add a slave if possible
+        if len(storagerouter_slave_ips) != 0:
+            node_ips.append(storagerouter_slave_ips[0])
+            HealthCheckCI.LOGGER.info("Added slave `{0}` to be checked by the healthcheck ..."
+                                      .format(storagerouter_slave_ips[0]))
+        else:
+            HealthCheckCI.LOGGER.warning("Did not add a slave to the list of to be tested nodes because "
+                                         "none were available")
+
+        # check if there are missing packages
+        for ip in node_ips:
+            HealthCheckCI.LOGGER.info("Starting the healthcheck on node `{0}`".format(ip))
+
+            missing_packages = SystemHelper.get_missing_packages(ip, HealthCheckCI.REQUIRED_PACKAGES)
+            assert len(missing_packages) == 0, "Missing {0} package(s) on `{1}`: {2}"\
+                .format(len(missing_packages), ip, missing_packages)
+
+            ########################################
+            # Testing command line, on remote node #
+            ########################################
+
+            client = SSHClient(ip, username="root")
+
+            assert client.run(["ovs", "healthcheck"]) is not None
+            assert client.run(["ovs", "healthcheck", "silent"]) is not None
+            assert client.run(["ovs", "healthcheck", "unattended"]) is not None
```

Comment:
if one of these asserts fail, we won't check the other ip's, perhaps we want to do that too ... and collect all errors?